python/housinginsights/ingestion/Cleaners.py

```diff
@@ -507,13 +508,32 @@ def clean(self, row, row_num=None):
 
         return row
 
-class topa_cleaner(CleanerBase):
+class TopaCleaner(CleanerBase):
+    def __init__(self, meta, manifest_row, cleaned_csv='', removed_csv='', engine=None):
+        #Call the parent method and pass all the arguments as-is
```

Comment:
Thanks @NealHumphrey.

I think the reason I did this was that if I created TopaCleaned.__init__ in order to pass the engine to the cleaner, it would skip CleanerBase.__init__. So the cleanest solution (at least, that I could think of) was to modify CleanerBase. A side benefit is that all classes inheriting from CleanerBase can now access the engine.